http4s/src/main/scala/com/avast/cloud/metrics/datadog/http4s/DatadogMetricsOps.scala

```diff
@@ -0,0 +1,63 @@
+package com.avast.cloud.metrics.datadog.http4s
+
+import java.time.Duration
+
+import cats.syntax.flatMap._
+import cats.effect.Sync
+import com.avast.cloud.metrics.datadog.api.{ MetricFactory, Tag }
+import org.http4s.{ Method, Status }
+import org.http4s.metrics.{ MetricsOps, TerminationType }
+
+object DatadogMetricsOps {
+
+  type ClassifierTags = String => List[Tag]
+
+  val defaultClassifierTags: ClassifierTags = classifier => List(Tag.of("classifier", classifier))
+
+  def make[F[_]](metricFactory: MetricFactory[F], classifierTags: ClassifierTags = defaultClassifierTags)(
```

Comment:
i wonder if possibility of adding `httpRequest => List[Tag]` possibility would be useful, in case someone wants to add for example tag with UserAgent or something like that

Reply:
Not possible, we don't have access to request itself.